src/modules/partition/core/PartitionLayout.cpp

```diff
@@ -144,6 +145,7 @@ PartitionLayout::addEntry( const QString& label,
     {
         entry.partFileSystem = m_defaultFsType;
     }
+    entry.partFeatures = features;
```

Comment:
This suggests to me that *features* should be part of the *PartitionEntry* constructor.

Reply:
Well, I guess this should apply to the attributes `partLabel`, `partMountPoint` and `partFileSystem`; but if so, the structure `PartitionEntry` is unless.

I preferred to left it as is. I guess it can be done in a second commit.

Reply:
Don't worry about this one: it applies polishing to other code, and doesn't have to be part of this MR. If you have time in your project, it's probably worth chasing some technical debt (like this one) but otherwise leave it and I'll get around to it eventually (that's what `TODO:` is for, right?)

Reply:
I can do it in another commit ;)

Furthermore, I would like to rename things. Like removing the `part` prefixes from the attributes that are redundant: `PartitionLayout::PartitionEntry::partLabel`. The `Partition` from `PartitionEntry` is redundant as well.

Also, the attribute `QString partLabel` should also be renamed to `QString partName` as it refers to the YAML attribute `PartitionLayout.name` in the file `partition.conf`.